src/software/backend/input/network/networking/network_client.cpp

```diff
@@ -129,59 +129,65 @@ void NetworkClient::filterAndPublishVisionData(SSL_WrapperPacket packet)
         world.updateFieldGeometry(field);
     }
 
-    if (packet.has_detection())
+    if (world.field().isFieldValid())
     {
-        SSL_DetectionFrame detection = *packet.mutable_detection();
-        bool camera_disabled         = false;
-
-        // We invert the field side if we explicitly choose to override the values
-        // provided by refbox. The 'defending_positive_side' parameter dictates the side
-        // we are defending if we are overriding the value
-        if (refbox_config->OverrideRefboxDefendingSide()->value() &&
-            refbox_config->DefendingPositiveSide()->value())
+        if (packet.has_detection())
```

Comment:
Do we need to nest these `if`'s or can we write it as `if (world.field().isFieldValid() && packet.has_dectection())`?

Reply:
We would need to nest them for `received_world_callback(world);` to be called when there's no detection but `Field` is valid